checker/jdk/index/src/java/lang/String.java

```diff
@@ -1735,7 +1735,7 @@ private int lastIndexOfSupplementary(int ch, int fromIndex) {
      * @param   target       the characters being searched for.
      * @param   fromIndex    the index to begin searching from.
      */
-    static @LTEqLengthOf("this") @SubstringIndexFor(value="this",offset="#1.length()-1") int indexOf(char[] source, int sourceOffset, int sourceCount,
+    static int indexOf(char[] source, int sourceOffset, int sourceCount,
```

Comment:
Is there a reason that these annotations are wrong, or are you just removing these because these methods aren't public?

Reply:
They are wrong - referring to `this` in a static method. Because the methods aren't public, I removed them rather than trying to fix them.

Reply:
You're right. I missed that these were static.